client_device_provisioning/fleet_provisoning/certs/root.ca.pem

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Is this certificate doing something? We may want to download the certificate as part of the instructions. If it's the Amazon root CA, then it should already be available in the CERTS_CONFIG_FOLDER (name needs checking!)